src/osyris/plot/wrappers.py

```diff
@@ -6,6 +6,8 @@
 from .. import config
 from ..core import Array
 from pint.quantity import Quantity
+import lic as lic
```

Comment:
The import of `lic` should be moved inside the `line_integral_convolution` function to avoid making `lic` a hard dependency for `osyris`